Solutions/Zscaler Private Access (ZPA)/Hunting Queries/ZscalerAbnormalTotalBytesSize.yaml

```diff
@@ -0,0 +1,31 @@
+id: 181dc982-4631-11ec-81d3-0242ac130003
+name: Zscaler - Abnormal total bytes size
+description: |
+  'Query shows abnormal total bytes size.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: ZscalerPrivateAccess
+    dataTypes:
+      - ZPAEvent
+tactics:
+  - Exfiltration
+  - Collection
+relevantTechniques:
+  - T1530
+  - T1537
+query: |
+  let Average_Body_Bytes = ZPAEvent
+  | where TimeGenerated > ago(14d)

```

Comment:
There is no real need to have a separate time length for the average calculation and the verification calculation.

Reply:
@spsocprime  - needs your review.

Reply:
fixed